app/plugins/Files/index.html

```diff
@@ -132,6 +138,39 @@
 				</div>
 			</div>
 		</div>
+		<!-- Sliding New Folder Frame -->
+		<div class='frame hidden' id='add-dir'>
+			<div class='header'>
+				<div class='button' id='back-dir'>
+					<i class='fa fa-chevron-left'></i>
+				</div>
+				<div class='title'>
+					Add Folder
+				</div>
+			</div>
+
+			<div class='pure-g step' id='step11'>
```

Comment:
This seems messy. How about the ids of the singular add-file steps be `filestep1` and the folder steps be `folderstep1`?